asapdiscovery-data/asapdiscovery/data/util/data_conversion.py

```diff
@@ -0,0 +1,26 @@
+from collections import defaultdict
+
+
+def get_first_value_of_dict_of_lists(d: dict[str, list]) -> dict[str, str]:
+    """
+    Get the first value of a dictionary of lists
+    """
+    return {k: v[0] for k, v in d.items()}
+
+
+def get_dict_of_lists_from_dict_of_str(d: dict[str, str], length=1) -> dict[str, list]:
```

Comment:
Can you make the docstring clearer with an example.

Reply:
added here: https://github.com/choderalab/asapdiscovery/pull/692/commits/5937b698832e894ca0331be347bbd116d7187cb2